lib/Serialization/ModuleFile.cpp

```diff
@@ -1465,7 +1617,8 @@ ModuleFile::ModuleFile(
     info.status = error(Status::Malformed);
     return;
   }
-
+  // Read source info file.
+  readModuleSourceInfoIfPresent();
```

Comment:
No error handling?

Reply:
Failing to load `.swiftsourceinfo` isn't a very big deal and probably is the default situation when loading a shipped Swift module. So I don't think it worths error handling.